<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QueryPatientWindow</class>
 <widget class="QWidget" name="QueryPatientWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QueryPatientWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
color: white; 
background-color: #151d20;
</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>481</width>
     <height>491</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="styleSheet">
       <string notr="true">border: 0px;</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>First Name:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Last Name:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_2"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="styleSheet">
       <string notr="true">QPushButton:hover {
background-color: #0c1416;
color: white;
font-size: 19px;
bottom-margin: 0px;
}

QPushButton {
background-color: #151d20;
font-size: 19px;
color: #657980;
border: 1px red;
}
</string>
      </property>
      <property name="text">
       <string>Query</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="styleSheet">
       <string notr="true">QPushButton:hover {
background-color: #0c1416;
color: white;
font-size: 19px;
bottom-margin: 0px;
}

QPushButton {
background-color: #151d20;
font-size: 19px;
color: #657980;
border: 1px red;
}
</string>
      </property>
      <property name="text">
       <string>Return</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
